2011-11-11  Ulrich Drepper  <drepper@gmail.com>

	[BZ #13147]
	* locales/es_BO: Use es_ES for LC_MESSAGES and LC_NUMERIC.
	* locales/es_CL: Likewise.
	* locales/es_CO: Likewise.
	* locales/es_DO: Likewise.
	* locales/es_EC: Likewise.
	* locales/es_GT: Likewise.
	* locales/es_HN: Likewise.
	* locales/es_MX: Likewise.
	* locales/es_NI: Likewise.
	* locales/es_PA: Likewise.
	* locales/es_PE: Likewise.
	* locales/es_PR: Likewise.
	* locales/es_PY: Likewise.
	* locales/es_SV: Likewise.
	* locales/es_UY: Likewise.
	* locales/es_VE: Likewise.
	* locales/es_ES: Fix LC_NUMERIC.
	* locales/es_CR: Use es_ES for LC_MESSAGES.

diff --git a/localedata/locales/es_BO b/localedata/locales/es_BO
index c77bb1d..6964311 100644
--- a/localedata/locales/es_BO
+++ b/localedata/locales/es_BO
@@ -56,8 +56,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -79,9 +78,7 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002C>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
diff --git a/localedata/locales/es_CL b/localedata/locales/es_CL
index ff44fa8..fe5b6d9 100644
--- a/localedata/locales/es_CL
+++ b/localedata/locales/es_CL
@@ -56,8 +56,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -79,9 +78,7 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002C>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
diff --git a/localedata/locales/es_CO b/localedata/locales/es_CO
index 4c1de13..5db5d6d 100644
--- a/localedata/locales/es_CO
+++ b/localedata/locales/es_CO
@@ -56,8 +56,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -79,9 +78,7 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002C>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
diff --git a/localedata/locales/es_CR b/localedata/locales/es_CR
index 6cff560..f998c86 100644
--- a/localedata/locales/es_CR
+++ b/localedata/locales/es_CR
@@ -47,8 +47,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
diff --git a/localedata/locales/es_DO b/localedata/locales/es_DO
index fcd0f9e..7cf54cf 100644
--- a/localedata/locales/es_DO
+++ b/localedata/locales/es_DO
@@ -56,8 +56,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -79,9 +78,7 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002E>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
diff --git a/localedata/locales/es_EC b/localedata/locales/es_EC
index b8d4c8f..480f6ef 100644
--- a/localedata/locales/es_EC
+++ b/localedata/locales/es_EC
@@ -56,8 +56,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -79,9 +78,7 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002C>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
diff --git a/localedata/locales/es_ES b/localedata/locales/es_ES
index fba4125..fcf90cb 100644
--- a/localedata/locales/es_ES
+++ b/localedata/locales/es_ES
@@ -85,8 +85,8 @@ END LC_MONETARY
 
 LC_NUMERIC
 decimal_point        "<U002C>"
-thousands_sep        ""
-grouping             0;0
+thousands_sep        "<U002E>"
+grouping             3;3
 END LC_NUMERIC
 
 LC_TIME
diff --git a/localedata/locales/es_GT b/localedata/locales/es_GT
index 455b076..49d369f 100644
--- a/localedata/locales/es_GT
+++ b/localedata/locales/es_GT
@@ -55,8 +55,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -78,9 +77,7 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002E>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
diff --git a/localedata/locales/es_HN b/localedata/locales/es_HN
index 53892b4..a809188 100644
--- a/localedata/locales/es_HN
+++ b/localedata/locales/es_HN
@@ -56,8 +56,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -79,9 +78,7 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002E>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
diff --git a/localedata/locales/es_MX b/localedata/locales/es_MX
index 91dac23..7a1cccc 100644
--- a/localedata/locales/es_MX
+++ b/localedata/locales/es_MX
@@ -56,8 +56,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -79,9 +78,7 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002E>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
diff --git a/localedata/locales/es_NI b/localedata/locales/es_NI
index c0c395f..207f2da 100644
--- a/localedata/locales/es_NI
+++ b/localedata/locales/es_NI
@@ -47,8 +47,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -70,9 +69,7 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002E>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
diff --git a/localedata/locales/es_PA b/localedata/locales/es_PA
index 10d6d82..ae5cd3a 100644
--- a/localedata/locales/es_PA
+++ b/localedata/locales/es_PA
@@ -56,8 +56,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -79,9 +78,7 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002E>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
diff --git a/localedata/locales/es_PE b/localedata/locales/es_PE
index 634c927..da1a640 100644
--- a/localedata/locales/es_PE
+++ b/localedata/locales/es_PE
@@ -56,8 +56,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -79,9 +78,7 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002E>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
diff --git a/localedata/locales/es_PR b/localedata/locales/es_PR
index 546b6c2..53496d4 100644
--- a/localedata/locales/es_PR
+++ b/localedata/locales/es_PR
@@ -47,8 +47,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -70,9 +69,7 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002E>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
diff --git a/localedata/locales/es_PY b/localedata/locales/es_PY
index 2c31b2f..ce226ba 100644
--- a/localedata/locales/es_PY
+++ b/localedata/locales/es_PY
@@ -56,8 +56,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -79,9 +78,7 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002C>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
diff --git a/localedata/locales/es_SV b/localedata/locales/es_SV
index 3b6be32..d99b6cf 100644
--- a/localedata/locales/es_SV
+++ b/localedata/locales/es_SV
@@ -56,8 +56,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -79,9 +78,7 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002E>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
diff --git a/localedata/locales/es_UY b/localedata/locales/es_UY
index 8b7ed0d..cbb7d5a 100644
--- a/localedata/locales/es_UY
+++ b/localedata/locales/es_UY
@@ -56,8 +56,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -79,9 +78,7 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002C>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
diff --git a/localedata/locales/es_VE b/localedata/locales/es_VE
index ca6637a..1ec2dff 100644
--- a/localedata/locales/es_VE
+++ b/localedata/locales/es_VE
@@ -56,8 +56,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -79,9 +78,7 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002C>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
